sunpy/data/_sample.py

```diff
@@ -77,41 +73,70 @@ def download_sample_data(progress=True, overwrite=True, timeout=None):
     -------
     None
     """
-    # Creating the directory for sample files to be downloaded
-    sampledata_dir = get_and_create_sample_dir()
-
-    number_of_files_fetched = 0
-    print("Downloading sample files to {}".format(sampledata_dir))
+    print("Downloading all sample files to {}. Overwriting if necessary.".format(sampledata_dir))
     for file_name in six.itervalues(_files):
-        if not overwrite:
-            if os.path.isfile(os.path.join(sampledata_dir,
-                                           file_name[0])):
-                number_of_files_fetched += 1
-                continue
-
-        for base_url in _base_urls:
-            full_file_name = file_name[0] + file_name[1]
+        get_sample_file(file_name, url_list=_base_urls, overwrite=True)
```

Comment:
I don't find where `_base_urls` is defined in this file. You deleted the old one, but the new one is only added in `sunpy/data/sample.py` but not imported in this one...